### PR TITLE
docs: runtime metrics

### PR DESCRIPTION
**Pull request by ZolotarevIgor-mail-ru**
Part of #124 
Detailed description of replication, fibers, runtime, operations and other metrics.

### CODE REVIEW

File: doc/monitoring/metrics_reference.rst [hunk 1]
```diff
@@ -6,6 +6,23 @@ Metrics reference
...
 
+-------------------------------------------------------------------------------
+General metrics
+-------------------------------------------------------------------------------
+
+General instance information.
+
+* ``tnt_cfg_current_time`` - instance system time in uxix timestamp format
+
+* ``tnt_cfg_listen`` - Tarantool port number
```
Comment (vasiliy-t) [inline doc/monitoring/metrics_reference.rst:17]: I suggest to not document metrics we are not collecting https://github.com/tarantool/metrics/pull/43/files

File: doc/monitoring/metrics_reference.rst [hunk 2]
```diff
@@ -122,3 +139,76 @@ Requests:
...
 
 * ``tnt_net_requests_current`` - amount of pending network requests
+
+.. _fibers:
+
+-------------------------------------------------------------------------------
+Fibers
+-------------------------------------------------------------------------------
+
+Provides statistics of fibers. If your app creates a lot of fibers, it can be used for monitoring
```
Comment (vasiliy-t) [inline doc/monitoring/metrics_reference.rst:149]: I suggest to add link to relevant docs sections, e.g. link to fibers

File: doc/monitoring/metrics_reference.rst [hunk 3]
```diff
@@ -122,3 +139,76 @@ Requests:
...
+Operations
+-------------------------------------------------------------------------------
+
+Number of iproto requests this instance has processed, aggregated by request type.
+It can be used to find out which type of operation clients make more often.
+
+Two types of metrics provided for each request type:
+
+* ``tnt_stats_op_total`` - total number of calls since server start
+* ``tnt_stats_op_rps`` - number of calls for the last 5 seconds
```
Comment (vasiliy-t) [inline doc/monitoring/metrics_reference.rst:172]: I suggest to not document _rps metrics at all, it's confusing and it's value could be derived from _total

File: doc/monitoring/metrics_reference.rst [hunk 4]
```diff
@@ -122,3 +139,76 @@ Requests:
...
+- ``insert`` - insert calls
+
+.. _replication:
+
+-------------------------------------------------------------------------------
+Replication
+-------------------------------------------------------------------------------
+
+Provides information of current replication status. To learn more about replication
+mechanism in Tarantool, see `this <https://www.tarantool.io/en/doc/2.6/book/replication/repl_architecture/#replication-mechanism>`_
```
Comment (vasiliy-t) [inline doc/monitoring/metrics_reference.rst:199]: we should discuss with docs team how not to hardcode links to internal docs sections.

**Review by vasiliy-t — CHANGES_REQUESTED** [review]

File: doc/monitoring/metrics_reference.rst [hunk 5]
```diff
@@ -6,6 +6,21 @@ Metrics reference
...
+General metrics
+-------------------------------------------------------------------------------
+
+General instance information.
+
+* ``tnt_cfg_current_time`` - instance system time in uxix timestamp format
+
+* ``tnt_info_lsn`` – LSN of instance
+
+* ``tnt_info_uptime`` – time since server was started, in seconds
```
Comment (opomuc) [inline doc/monitoring/metrics_reference.rst:19]: server or instance?

File: doc/monitoring/metrics_reference.rst [hunk 6]
```diff
@@ -122,3 +137,73 @@ Requests:
...
+-------------------------------------------------------------------------------
+Fibers
+-------------------------------------------------------------------------------
+
+Provides statistics of :ref:`fibers <fiber-fibers>`. If your app creates a lot of fibers, it can be used for monitoring
+fibers count and memory usage.
+
+* ``tnt_fiber_count`` - number of fibers
+
+* ``tnt_fiber_csw`` - averall fibers context switches number
```
Comment (opomuc) [inline doc/monitoring/metrics_reference.rst:152]:
```suggestion
* ``tnt_fiber_csw`` - averall amount of fibers context switches
```

**Review by vasiliy-t — APPROVED** [review]

**Review by opomuc — APPROVED** [review]

**Pull request merged**